tests/test_isrTaskLSST.py

```diff
@@ -292,14 +293,16 @@ def test_isrBootstrapFlat(self):
     def test_isrBias(self):
         """Test processing of a bias frame."""
         mock_config = self.get_mock_config_no_signal()
+        # mock_config.doAddSerialCti = False
 
         mock = isrMockLSST.IsrMockLSST(config=mock_config)
         input_exp = mock.run()
 
         isr_config = self.get_isr_config_electronic_corrections()
         isr_config.doBias = True
-        # We do not do defect correction when processing biases.
+        # We do not do these corrections when processing biases.
         isr_config.doDefect = False
+        isr_config.doDeferredCharge = False
```

Comment:
In regular bias processing (*not* bootstrap) we will be doing deferred charge correction.